app/lib/imports/missing_journey_ending_events.rb

```diff
@@ -25,16 +25,17 @@ def call
 
   attr_reader :csv_path, :column_mapper
 
-  ALLOWED_JOURNEY_STATES = %w[Completed Cancelled].freeze
+  ALLOWED_EXISTING_JOURNEY_STATES = %w[proposed in_progress].freeze
```

Comment:
Is there test coverage for this?

Reply:
Although not explicit, it's covered here: https://github.com/ministryofjustice/hmpps-book-secure-move-api/pull/1688/files#diff-51febc3e46a5dfffe774de9c450420b67347e50e4cbc062a6c4800ec34a98eb4R8